.github/workflows/test-build.yml

```diff
@@ -329,6 +329,37 @@ jobs:
             -f "$REPO_DIR_NAME/docker-compose.e2e.yml" \
             -f docker-compose.all.yml \
             up --exit-code-from cypress
+      
+      - name: Run docker-compose stfc
+        env:
+          SVC_ACC_TOKEN: ${{ secrets.SVC_ACC_TOKEN }}
+        run: |
+          cd "$GITHUB_WORKSPACE/.."
+
+          REPO_DIR_NAME=$(basename $GITHUB_WORKSPACE)
+
+          export USER_OFFICE_BACKEND_TAG=${{ needs.resolve_dep.outputs.BE_TAG }}
+          export USER_OFFICE_BACKEND_DIR=user-office-backend
+          export USER_OFFICE_BACKEND=http://backend:4000/graphql
+          export USER_OFFICE_ENDPOINT=$USER_OFFICE_BACKEND
+
+          export USER_OFFICE_FACTORY_TAG=${{ needs.resolve_dep.outputs.FACTORY_TAG }}
+          export USER_OFFICE_FACTORY_ENDPOINT=http://factory:4500/generate
+
+          export USER_OFFICE_FRONTEND_DIR=$REPO_DIR_NAME
+          export BUILD_TAG=${{ github.sha }}
+
+          export CYPRES_SPEC_PATTERN=${{ matrix.pattern }}
```

Comment:
Is this meant to be cypres instead of cypress?

Reply:
We call it in the `docker-compose.yml` file like like: `cypress run --spec ${CYPRES_SPEC_PATTERN}` so it gets mapped correctly. The e2e config spells it the same so I just kept it the same as that.

Reply:
Cool, just double checking 👍